source/includes/gcp/_k8_deployments.md

```diff
@@ -0,0 +1,157 @@
+### Deployments
```

Comment:
Going along with what Milt suggested for the other file, should this then be `## Deployments`?

Reply:
Where the other file was Kubernetes->Deployments, this one is GCP->Kubernetes->Deployments, so this needs the extra #